wp-pgp-encrypted-emails.php

```diff
@@ -1295,16 +1247,17 @@ private static function prepareMail ($to, $subject, $message, $headers, $attachm
                     $headers = $parts[0];
                 }
 
-            		//Unlink the temporary files:
-            		unlink($clearfile);
-            		unlink($encfile);
-
           } catch (Exception $e) {
                 error_log(sprintf(
                     __('Cannot send encrypted email to %1$s', 'wp-pgp-encrypted-emails'),
                     $to
                 ));
             }
+
+            //Unlink the temporary files:
```

Comment:
Better, thanks! Now that this outside the `try`/`catch` block we no longer need this comment since these lines are *always* executed, unconditionally.